Elastic: Internal data links (#25942)
grafana/grafana: docs/sources/features/datasources/elasticsearch.md

- **Message field name:**  message
- **Level field name:** fields.level

> ### Data links

Reviewer: This is going to be very confusing, considering we have a feature of the same name that works differently than this. @dprokop, thoughts?

Author: This feature is actually fairly similar and we will probably unify it even more in the future so not sure if it make sense to change the name right now. There is a difference in with Loki where we do data link + create a new field that's why the naming there is different.

> Data links create a link from specified field that can be accessed in logs view in Explore.

Reviewer: ```suggestion
Data links create a link from a specified field that can be accessed in logs view in Explore.
```

Each data link configuration consists of: 

> - **Field**: Name of the field used by the data link

Reviewer: ```suggestion
- **Field -** Name of the field used by the data link.
```

> - **Url/Query**: If link is external, you fill in full url for the link. In case of internal link this input serves as query for the target data source. In both cases you can interpolate the value from the field with `${__value.raw }` macro.

Reviewer: ```suggestion
- **URL/query -** If the link is external, then enter the full link URL. If the link is internal link, then this input serves as query for the target data source. In both cases, you can interpolate the value from the field with `${__value.raw }` macro.
```

> - **Internal link**: Switch to select if link is internal or external. In case of internal link, data source selector will be shown where you can select the target data source. At this moment only tracing data sources are supported.

Reviewer: ```suggestion
- **Internal link -** Select if the link is internal or external. In case of internal link, a data source selectorallows you to select the target data source. Only tracing data sources are supported.
```
